### ISSUE TITLE
I can’t do “undefine” for a virtual machine with uefi nvram

### ISSUE THREAD

**Issue by stm32repo** [ansible-collections/community.libvirt#159]
##### SUMMARY
I can’t do “undefine” for a virtual machine with uefi nvram
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME
community.libvirt
##### ANSIBLE VERSION
core 2.14.3

```
 <os>
    <type arch='x86_64' machine='pc-q35-7.2'>hvm</type>
    <loader readonly='yes' secure='yes' type='pflash'>/usr/share/OVMF/OVMF_CODE_4M.fd</loader>
    <nvram template='/usr/share/OVMF/OVMF_VARS.fd' type='file'>
      <source file='/var/lib/libvirt/nvram/ansiblevm-1.fd'/>
    </nvram>
    <boot dev='hd'/>
  </os>
```

Error: `An exception occurred during task execution. To see the full traceback, use -vvv. The error was: libvirt.libvirtError: Requested operation is not valid: cannot undefine domain with nvram
failed: [127.0.0.1] (item=1) => {"ansible_loop_var": "item", "changed": false, "item": "1", "msg": "Requested operation is not valid: cannot undefine domain with nvram"}`

I can manually delete the VM using the command:
`virsh --connect qemu:///system undefine --nvram ansiblevm-1`

**Comment by csmart** [comment]
Hi @stm32repo, thank you for the report, I will look into it and see if we can find a way to fix it. Thanks!

**Comment by bturmann** [comment]
This issue was fixed in #136 and is included in 1.3.0 which was released just a few hours ago. Galaxy is already updated. The docs are still showing 1.2.0, not sure when they get updated.
